Themes/default/Register.template.php

```diff
@@ -560,6 +568,18 @@ function template_admin_register()
 						<dd>
 							<input type="checkbox" name="emailActivate" id="emailActivate_check" tabindex="', $context['tabindex']++, '"', !empty($modSettings['registration_method']) && $modSettings['registration_method'] == 1 ? ' checked' : '', ' onclick="onCheckChange();">
 						</dd>
+						<dt>
+							<strong><label for="requireAgreement">', $txt['admin_register_require_agreement'], ':</label></strong>
+						</dt>
+						<dd>
+							<input type="checkbox" name="requireAgreement" id="requireAgreement_check" tabindex="', $context['tabindex']++, '" checked="checked" disabled="disabled" class="input_check" /><input type="hidden" name="requireAgreement" value=1>
+						</dd>
+						<dt>
+							<strong><label for="requirePolicyAgreement">', $txt['admin_register_require_policy'], ':</label></strong>
+						</dt>
+						<dd>
+							<input type="checkbox" name="requirePolicyAgreement" id="requirePolicyAgreement_check" tabindex="', $context['tabindex']++, '" checked="checked" disabled="disabled" class="input_check" /><input type="hidden" name="requirePolicyAgreement" value=1>
+						</dd>
```

Comment:
I think we can leave out these checkboxes in 2.1, and instead just hard code the behaviour in Register.php so that newly created users will always be presented with the agreement acceptance form when they first log in.